### PR TITLE
Adds the Security Assistant job 

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
This PR adds the security assistant, a lower access and lower stakes security role that allows newer players to learn and get acclimated to security play without being bombarded with a bunch of responsibilities. The role emphasizes communication and investigation skills, rather than combat. 

The security assistant role has a minimal timelock (5 rounds) to serve as a very low barrier of entry, in order to encourage newer players to try out the role. However, the timelock for regular security officers has been raised (30 rounds, though this number can be debated and lowered if need be) so that people's introductions to security aren't so high pressure. There are two security assistant slots every round, which should hopefully enable a synergy of officers teaching assistants and trainees assisting officers where they can, without either party getting too overwhelmed. 

The security assistant: 
- Receives a special security token that is redeemable for a barrier, a forensic scanner, and a special security ticket writer device (sprite by Flaborized!)
- Receives a special security pouch with two pairs of handcuffs, two flashes, a whistle, and a frosted donut
- Spawns with a special security assistant jumpsuit (sprite by Flaborized!), fingerless gloves, and brown shoes
- Like regular security officers, spawns with security trainee, a clone disk, a badge, secHUDs, and a security headset
- Like regular security officers, cannot be an antag, and has been blacklisted from gamemodes such as gang and rev
- Has access to the security department, the brig, maintenance tunnels, and weapon and contraband permits
- Has lower pay
- Unlocks a special medal when they escape alive to Centcom

This PR also:
- Adds a paragraph on guidance for security assistants and their expected responsibilities in Space Law book
- Adds a special security alert module that now spawns in all security PDAs. When used, this module sends a "backup requested" PDA alert to the entire security department from the user's PDA
- Adds the special security ticket writer device to all maps' security departments
- Cleans up some weapons vendor code

## ToDo
These are thing I hopefully want to do but would greatly appreciated if others assisted.
- Add wiki page for security assistants
- Add possible Security For Beginners guide book

## Why's this needed? 
To provide an entry point to players interested in the security department without starting off in a high pressure job like security officer, while also expanding the security department to be more of a department and less 5 dudes with electrified sticks governed by a guy in a cape with a speaking gun.

Changelog will be updated to reflect what changes are currently considered finished
## Changelog

```
(u)Carbadox, Flaborized, and Flourish
(*)Added a new permanent security role, security assistant. This role will provide an entry point to players interested in the Security department without having to start off in a high pressure role like security officer. Please be nice to them!
(*)Security PDAs now spawn with a special security alert module. When used, this module sends a "backup requested" PDA alert to the entire security department from the user's PDA.  
(*)Security departments now have a couple ticket writer devices laying around, which allow officers to issue tickets (and only tickets) without having to wrangle with their PDA.  
```

### CODE REVIEW

**Comment by Carbadox** [comment]
I'll add an input wanted label when I take off the WIP label. I want forum feedback to be in regards to a completed PR

**Comment by pali6** [comment]
You can also convert the PR to a draft if it's still WIP (top right under reviewers).

**Comment by MatheusIsNotAvailable** [comment]
I think 50 rounds might be a bit too harsh, 25 would be a better ammount imo.

I don't see why they would need that much access, aren't they just supposed to take care of prisoners, and learn from watching sec officers(idk if they have it right now but they shouldn't have access to security lockers equipment too)?

The number of trainees shouldn't be the same as the ammount of officers too, just half the ammount of the sec team, otherwise it would end up crowding the place to much.

apart from these 3 things I see no problems with it, it will also be good for sec officers to have something to do that isn't related to chasing after criminals. (How does Flabo's "nearly security trainee" look btw? ideally it should be pretty distinct from normal sec jumpsuits)

**Comment by jan-antilles** [comment]
What kind of tools do Security Trainees get? SecHUD glasses? Flash? I would want to make sure they can defend themselves at least a little, since they will definitely be picked on, but I also don't want them to be viewed as easy pickings for getting Sec gear. Also, since the recent packet changes have made secpass codes highly desirable to packet hackers: do the Trainees have clone disks?

From an RP perspective, I love this change. Especially pairing it with increasing the round requirement for Secoffs proper. I think this will help us mold a culture of Security that focuses on teaching and interacting more with the rest of the station. 

Love it!!

**Comment by Tarmunora** [comment]
why no sec training trait?

**Comment by TTerc** [comment]
don't forget to add them to gang and rev blacklist so we don't have vice officer situation again
all in all, I like the idea

**Comment by flrsh** [comment]
Ignore checks. Gonna redo the maps after finalizing feedback, right before merging.

**Comment by Carbadox** [comment]
one thing to note is that whistles are currently slightly broken, where they won't show the "X blows the whistle" text. probably worth fixing as well

File: maps/kondaru.dmm [hunk 1]
```diff
@@ -42635,6 +42646,21 @@
 /obj/cable{
 	icon_state = "2-8"
 	},
+<<<<<<< HEAD
```
Comment (Carbadox) [inline maps/kondaru.dmm:42649]: This might be why checks are failing

**Comment by pali6** [comment]
I don't think I can really judge this PR at all. But I saw some neat ticket typewriter sprites a while ago and I think it'd be great if this job maybe started with those? (This post was sponsored by the committee for improving monkey AI and letting them write stuff on typewriters.)

**Comment by maxymax13** [comment]
assistants were originally supposed to assist sec, you have come full circle

**Pull request merged**